example/conf/routes

```diff
@@ -1,6 +1,6 @@
 GET        /                      controllers.Application.index
 GET        /authenticated         controllers.Application.authenticated
-GET        /authenticatedGroup    controllers.Application.authenticatedAndInGroup
+#GET        /authenticatedGroup    controllers.Application.authenticatedAndInGroup
```

Comment:
An oversight, initially just commented this out to get things to compile, will fix.